content/docs/04.user-interface-guide/04-executions.md

```diff
@@ -10,4 +10,19 @@ When clicking on an execution id or on the eye icon, you can open the page of an
 
 An **Execution** page will allow access to the details of a flow execution, including logs, outputs, and metrics. 
 
-![Kestra User Interface Execution Page](/docs/user-interface-guide/09-Executions-Execution.png)
\ No newline at end of file
+![Kestra User Interface Execution Page](/docs/user-interface-guide/09-Executions-Execution.png)
+
+
+### Outputs
+
+The outputs in an execution page allow to see each task outputs.
```

Comment:
```suggestion
The outputs tab in an execution page allow to see each task outputs.
```

Let's call it `tab`, we already use it somewhere else

Reply:
done